arbitrum-docs/how-arbitrum-works/bold/gentle-introduction.md

```diff
@@ -12,148 +12,266 @@ import PublicPreviewBannerPartial from './partials/_bold-public-preview-banner-p
 
-## What _exactly_ is BOLD?
+## What _exactly_ is BoLD?
 
-BOLD, an acronym for Bounded Liquidity Delay Protocol, is an upgrade to Arbitrum's existing dispute protocol. Specifically, BOLD changes some of the rules used by validators to open and resolve disputes about Arbitrum’s state to ensure only valid states get confirmed on an Arbitrum chain’s parent chain, such as Ethereum.
+BoLD, an acronym for Bounded Liquidity Delay Protocol, is an upgrade to Arbitrum's existing dispute protocol. Specifically, BoLD changes some of the rules used by validators to open and resolve disputes about Arbitrum’s state to ensure only valid states get confirmed on an Arbitrum chain’s parent chain, such as Ethereum.
 
-The current dispute protocol has working fraud proofs and is used in production today by Arbitrum chains. The changes BOLD brings enable anyone to participate in the validation of the state of the chain and enhance security around withdrawals to L1.
+The current dispute protocol has working fraud proofs and is used in production today by Arbitrum chains. The changes BoLD brings enable anyone to participate in the validation of the state of the chain and enhance security around withdrawals to L1.
 
 A bonded validator’s responsibilities are to:
 
-- Post claims about an Arbitrum chain’s state to Ethereum,
-- Challenge invalid claims made by other validators, and
-- Confirm valid claims - either by timing other validators out or by winning a challenge
+- Post claims about an Arbitrum chain’s state to its parent chain (for Arbitrum One, the parent chain is L1 Ethereum),
+- Open challenges to dispute invalid claims made by other validators, and
+- Confirm valid claims by participating in and winning challenges.
 
-The goal of BOLD is to unlock permissionless validation by ensuring that disputes are resolved within a fixed period of time (currently equivalent to 2 challenge periods, plus a small grace period for the Security Council and a small delta for computation), effectively removing the risk of delay attacks and making withdrawals to the parent chain more secure. BOLD accomplishes this by introducing a new dispute system that lets any, one entity defend Arbitrum against malicious parties - effectively allowing anyone to validate Arbitrum’s state without needing permission to do so.
+The goal of BoLD is to unlock permissionless validation by ensuring that disputes are resolved within a fixed period of time (currently equivalent to 2 challenge periods, plus a 2 day grace period for the Security Council to intervene if necessarym, and a small delta for computation), effectively removing the risk of delay attacks and making withdrawals to a parent chain more secure. BoLD accomplishes this by introducing a new dispute system that lets any, single entity defend Arbitrum against malicious parties - effectively allowing anyone to validate, propose, and defend an Arbitrum chain’s state without needing permission to do so.
```

Comment:
The goal of BoLD is to unlock permissionless validation by ensuring that disputes are resolved within a fixed period (currently equivalent to 2 challenge periods, plus a two-day grace period for the Security Council to intervene if necessary and a small delta for computation), effectively removing the risk of delay attacks and making withdrawals to a parent chain more secure. BoLD accomplishes this by introducing a new dispute system that lets any single entity defend Arbitrum against malicious parties - effectively allowing anyone to validate, propose, and defend an Arbitrum chain’s state without needing permission to do so.